src/data/modifiers.txt

```diff
@@ -3621,6 +3621,8 @@ Item	the Ley Incursion's waist	Mysticality: +15, Mysticality Percent: +25, Exper
 Item	The Lost Glasses	Reduce Enemy Defense: 15, Hot Resistance: +2, Single Equip
 Item	The One Mood Ring	Muscle Percent: [5*G], Mysticality Percent: [5*G], Moxie Percent: [5*G], Monster Level: [25-5*G], Single Equip
 Item	The Ring	Muscle: +3, Mysticality: +3, Moxie: +3, Spooky Damage: +40, Single Equip
+# Item The Superconductor's CPU: (based on Crimbo 22 Elf Gratitude)
+Item	The Superconductor's CPU	Maximum HP: [floor(sqrt(pref(elfGratitude)))], Maximum MP: [floor(sqrt(pref(elfGratitude)))]
```

Comment:
Does this cause issues if the gratitude pref has commas in it?

Reply:
It does not.

All Preferences are strings.
Preferences.getInteger("elfGratitude") passes it through StringUtilities.parseInteger, which handles commas.
So does ash get_property("elfGratitude").to_integer()

Reply:
```
> get elfGratitude

2,335

> ash string_modifier($item[The Superconductor's CPU], "Evaluated Modifiers")

Returned: Maximum HP: +48, Maximum MP: +48
```